Description: on kFreeBSD python atexit will make mess, disable it for now
Origin: upstream, https://github.com/unbit/uwsgi/commit/b19074
Author: Unbit <info@unbit.it>
Reviewed-By: Jonas Smedegaard <dr@jones.dk>
Forwarded: yes
Last-Update: 2013-09-25

diff --git a/plugins/python/python_plugin.c b/plugins/python/python_plugin.c
index aeaa9eb..acba17a 100644
--- a/plugins/python/python_plugin.c
+++ b/plugins/python/python_plugin.c
@@ -308,6 +308,10 @@ realstuff:
 	// this time we use this higher level function
 	// as this code can be executed in a signal handler
 
+#ifdef __FreeBSD_kernel__
+	return;
+#endif
+
 	if (!Py_IsInitialized()) {
 		return;
 	}
